app/models/user.rb

```diff
@@ -1,8 +1,12 @@
 class User < ActiveRecord::Base
   authenticates_with_sorcery!
 
-  validates :email, :password, presence: true
+  validates :email, presence: true
   validates :email, uniqueness: true, if: -> { email.present? }
+  validates :password, presence: true, on: :update, confirmation: true
```

Comment:
I believe password should also be present on create (Onboarding flow). Also, I'm not a big fan of the password confirmation box. I think instead of confirming a password, we should have a password forget. (I like clean forms :D )